### ISSUE TITLE
Warning: Breach of contract warning letter

### ISSUE THREAD

**Issue by airdrop1111111** [lbkolev/terraform-aws-eth-node#9]
I don't understand why you would lie to me for $20, you are so poor that you are crazy? Besides, do you really think I only have one github account? I can have countless accounts if I want, and this is just one of them.
I have taken a snapshot of your watch list and followed list. I will use ten different email addresses to send you and anyone who has any contact with you every day. In addition, I will also use ten different email addresses every day. The newly registered github account will leave debt collection information under all your warehouses.
To sum up, if you are willing to bear the price to cheat me $20, please feel free to do so. Anyway, I have automated procedures for doing this. It does not require any times on my part.

**Comment by airdrop1111111** [comment]
so?
